tempodb/metrics/metrics.go

```diff
@@ -0,0 +1,39 @@
+package metrics
```

Comment:
discussed elsewhere, but i do like the tight coupling of metrics with the code that uses them in the same file.

i consider this a minor preference though and can go either way.

Reply:
Yeah agree about the tight coupling, but didn't come up with a convincing refactor that kept it.  The way I chose is for the proto and future (parquet) compactor implementations to have direct access to these metrics, but this also means moving the metrics to avoid a circular dependency.  I.e. tempodb compactor (metrics originally here) -> encoding -> v2 -> metrics.   Otherwise, if we keep the metrics in tempodb/compactor.go, then we have to devise something like callbacks for each time an object is combined, or bytes written.  That seemed less readable than moving the metrics.